package.json

```diff
@@ -12,13 +12,15 @@
     "type": "git",
     "url": "https://github.com/enactjs/eslint-config-enact.git"
   },
+  "eslintConfig": {
+    "extends": "./index.js"
+  },
   "peerDependencies": {
     "babel-eslint": "10.x",
-    "eslint": "5.x",
+    "eslint": "6.x || 7.x",
```

Comment:
I wonder if we can support 6.x with this?  Maybe better to keep 5.x and 7.x?  Maybe with the other changes we can?

Reply:
We may wish to set 7.x frankly, and remove the deprecated rules, to be honest.  I left it at 6.x+ to support last 2 major releases, which cover until roughly a year ago. If we need to support 5.x onward we can if needed though.

Reply:
I'd be fine with that, I guess?  That lets us adopt new rules, too.